ui/js/controllers/jobs.js

```diff
@@ -71,6 +71,10 @@ treeherderApp.controller('JobsCtrl', [
             }
         };
 
+        if($location.search().revision === 'undefined'){
```

Comment:
There is a code, just below this code, that uses if without the spaces, should I modify it too??